content-services/latest/develop/sdk.md

```diff
@@ -2560,7 +2511,7 @@ So, taking into account the previous section, let's see how the integration test
                         <artifactItem>
                             <groupId>org.apache.httpcomponents</groupId>
                             <artifactId>httpclient</artifactId>
-                            <version>4.5.2</version>
+                            <version>4.6.2</version>
```

Comment:
I'm not sure if this version number is related to the SDK version - please check

Reply:
Has been done in https://github.com/Alfresco/docs-alfresco/pull/1074